### PR TITLE
fix: network modal missing version

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

The network `version` value wasn't being saved in previous wallet versions. This is a patch to re-populate the `version` from the network.

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: src/renderer/components/Network/NetworkModal.vue [hunk 1]
```diff
@@ -346,7 +347,29 @@ export default {
...
+
+        if (networkConfig) {
+          this.form.version = networkConfig.version.toString()
+        } else {
+          try {
+            ClientService.fetchNetworkConfig(this.form.server)
+              .then(network => {
+                console.log('fetchNetworkConfig')
+                this.form.version = network.version.toString()
+                this.$error('Address Version was missing - please check your wallets')
```
Comment (luciorubeens) [inline src/renderer/components/Network/NetworkModal.vue]: Maybe move this to the locale file and remove debugging.

Comment (alexbarnsley) [reply src/renderer/components/Network/NetworkModal.vue]: Yes you're right 🤔 will do it a bit later on. Thanks 👌

**Comment by alexbarnsley** [comment]
oops

File: src/renderer/components/Network/NetworkModal.vue (hunk 1 above)
Comment (alexbarnsley) [reply src/renderer/components/Network/NetworkModal.vue]: Done @luciorubeens

**Pull request merged**